docs/tutorials/content.md

```diff
@@ -0,0 +1,155 @@
+## ITowns' basics
+
+### 1 - Introduction
+
+### 2 - Data visualization with iTowns
+
+- The support to display anything in iTowns is a `View`. 
+- Every geographic data are supported by a `Layer`, which is attached to the `View`.
+- A `Camera`, which is also attached to the `View`, provides an interface between what is visualized on the `View` and the user.
+- This `Camera` can be steered thanks to `Controls`.
+  
+
+- A view supports a unique CRS (which can be different from one `View` to another).
```

Comment:
The view displays 3d data in a unique CRS. But it's different for the Layers and data Source type (vector or raster).